README.md

```diff
@@ -338,6 +338,12 @@ Unit tests for this package are written with [Jest](https://jestjs.io/en/). To r
 
 Javascript linting is enforced using the [Springer Nature Eslint config](https://www.npmjs.com/package/@springernature/eslint-config). Run the linter using `npm run lint` from the command line.
 
+## Dependency graph
+
+Visual graph of module dependencies generated using [Madge](https://www.npmjs.com/package/madge)
+
+<img src="img/graph.svg" width="914">
+
```

Comment:
Might be worth while adding the "Gives a visual aid for working on the codebase." there too